website/docs/core-concepts/data-access-and-binding/capturing-data-write/README.md

```diff
@@ -39,10 +39,10 @@ In the examples above, **`text`** is the internal property of the **`nameInput`*
 
 ## Triggering Updates
 
-Since write operations are more expensive, the Query should be triggered once all the user data is captured. To do this, we can make use of a [Button](../../../reference/widgets/button/) widget and configure the [Query](../querying-a-database/) to run in the onClick of the [button](../../../reference/widgets/button/).
+Since write operations are more expensive, the Query should be triggered once all the user data is captured. To do this, we can make use of a [Button](/reference/widgets/button/) widget and configure the [Query](/core-concepts/data-access-and-binding/querying-a-database/) to run in the onClick of the [button](/reference/widgets/button/).
```

Comment:
```suggestion
Since write operations are more expensive, the Query should be triggered once all the user data is captured. To do this, we can make use of a [Button](/reference/widgets/button/) widget and configure the [Query](/core-concepts/data-access-and-binding/querying-a-database) to run in the onClick of the [button](/reference/widgets/button).
```